pandas/core/indexes/period.py

```diff
@@ -393,7 +392,7 @@ def is_full(self) -> bool:
         if not self.is_monotonic_increasing:
             raise ValueError("Index is not monotonic")
         values = self.asi8
-        return ((values[1:] - values[:-1]) < 2).all()
+        return ((values[1:] - values[:-1]) < 2).all().item()
```

Comment:
why is this needed?  seems like the kind of thing that might be fragile across numpy versions

Reply:
The function is annotated as returning bool, but it currently return numpy.bool_ (which does not seem to inherit from bool):

```py
>>> type((np.random.rand(3) > 0.5).all())
<class 'numpy.bool_'>
>>> isinstance((np.random.rand(3) > 0.5).all(), bool)
False
```